src/main/java/org/reaktivity/nukleus/tls/internal/TlsController.java

```diff
@@ -196,4 +196,9 @@ public String name()
                      .build()
                      .sizeof();
     }
+
+    public long count(String name)
+    {
+        return controllerSpi.doCount(name);
+    }
```

Comment:
Suggest switching to `framesRead(long routeId)`, `bytesRead(long routeId)`, `framesWritten(long routeId)` and `bytesWritten(long routeId)` methods.